### ISSUE TITLE
The app did work but now it do not show up (it actually opens)

### ISSUE THREAD

**Issue by neotherack** [anujdatar/onenote-desktop#77]
The app was working until recently but now it just do not show up when I click on it.

After that, I did just theck "ps -aux  | grep -i onenote", I can see it running but there is not window for it at all.
Then, I just tried to run it from terminal just by issuing "onenote-desktop".

I see this:
![image](https://github.com/anujdatar/onenote-desktop/assets/17471672/c8e41cd6-92f9-45b2-bacc-b5ea46b1c29b)

Error msg:
A JavaScript error occurred in the main process
Uncaught Exception:
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Conf._deserialize (/snap/onenote-desktop/20/resources/app.asar/node_modules/conf/dist/source/index.js:67:43)
    at get store [as store] (/snap/onenote-desktop/20/resources/app.asar/node_modules/conf/dist/source/index.js:278:43)
    at new Conf (/snap/onenote-desktop/20/resources/app.asar/node_modules/conf/dist/source/index.js:131:32)
    at new ElectronStore (/snap/onenote-desktop/20/resources/app.asar/node_modules/electron-store/index.js:69:3)
    at Object.<anonymous> (/snap/onenote-desktop/20/resources/app.asar/lib/appConfig.js:21:16)
    at Module._compile (node:internal/modules/cjs/loader:1269:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1324:10)
    at Module.load (node:internal/modules/cjs/loader:1124:32)
    at Module._load (node:internal/modules/cjs/loader:965:12)

Did anyone experience that?
I'm on Ubuntu 22.04.4 LTS and installed it using snap.

**Comment by anujdatar** [comment]
Oh, it looks like the config (json) file might be corrupted. Could you try deleting the config file for me and then launch the app? I think that sould fix the issue.

Since you are on Ubuntu and using Snap, the following steps should fix it. This just clears the basics like window placement, size, and last known location. And, keeps all other data intact.
1. Close the app.
2. Go to `/home/<user>/snap/onenote-desktop/current/.config/OneNote/`.
3. Delete the `config.json` file.
4. Restart the app.

If you would like to completely reset the app (i.e.settings, local history, cache and login session). This should not be required, but just as an FYI.
1. Close the app
2. Delete the entire folder mentioned in **Step 2** above.
3. Restart the app.

**Comment by anujdatar** [comment]
Once the app opens, you can actually do both these things from within the app menu, but since your app isn't launching these should be the easiest way to do it.

I should probably add a check for malformed `config.json` file, and have the app handle automatic reset. I'll add that to the todo list.

Anyway, let me know if this works, or if the issue persists.
Thanks,
Anuj

**Comment by neotherack** [comment]
FIxed! 
I did rename the config.json to old_config.json, then started the app and it worked perfect.

After that, I did check the contents of the old config.json file, it was just empty (0 bytes).
Just after launching the app, a new config.json showed up and it has now some contents.

![image](https://github.com/anujdatar/onenote-desktop/assets/17471672/d87fd146-b717-41a9-8887-332ac25eb1e2)


Thank you!
